arviz/plots/posteriorplot.py

```diff
@@ -62,8 +63,11 @@ def plot_posterior(
     textsize: float
         Text size scaling factor for labels, titles and lines. If None it will be autoscaled based
         on figsize.
-    credible_interval: float, optional
-        Credible intervals. Defaults to 0.94. Use None to hide the credible interval
+    hpd_interval: float, optional
+        Plots highest posterior density interval for chosen percentage of density.
+        Use 'hide' to hide the credible interval. Defaults to 0.94.
+    hpd_interval: float or str, optional
+        Credible intervals. Defaults to 0.94. Use 'hide' to hide the hpd interval
```

Comment:
docs got duplicated here

Reply:
There are also other docstring where description of hpd_interval says "Credible interval...".